docs/src/literate-tutorials/linear_elasticity.jl

```diff
@@ -1,3 +1,354 @@
+        dΩ = getdetJdV(cellvalues, q_point)
+        for i in 1:getnbasefunctions(cellvalues)
+            ## Gradient of the test function
+            ∇Nᵢ = shape_gradient(cellvalues, q_point, i)
+            for j in 1:getnbasefunctions(cellvalues)
+                ## Symmetric gradient of the trial function
+                ∇ˢʸᵐNⱼ = shape_symmetric_gradient(cellvalues, q_point, j)
+                ke[i, j] += (∇Nᵢ ⊡ ∂σ∂ε ⊡ ∇ˢʸᵐNⱼ) * dΩ
+            end
+        end
+    end
+    return ke
+end
+#md nothing #hide
+#
+#md # !!! details "Performance tips"
+#md #     1. In this tutorial, `∂σ∂ε`, is minor symmetric, and we could have used
+#md #        the symmetric part of test function gradient above.
+#md #     2. The product `∇Nᵢ ⊡ ∂σ∂ε` can be done outside the "j"-loop, but is kept inside here
+#md #        to highlight the similarity to the finite element form.
+#md #
+# ### Global assembly
+# We define the function `assemble_global` to loop over the elements and do the global
+# assembly. The function takes the preallocated sparse matrix `K`, our DofHandler `dh`, our
+# `cellvalues` and the elastic stiffness tensor `∂σ∂ε` as input arguments and computes the
+# global stiffness matrix `K`.
+function assemble_global!(K, dh, cellvalues, ∂σ∂ε)
+    ## Allocate the element stiffness matrix
+    n_basefuncs = getnbasefunctions(cellvalues)
+    ke = zeros(n_basefuncs, n_basefuncs)
+    ## Create an assembler
+    assembler = start_assemble(K)
+    ## Loop over all cells
+    for cell in CellIterator(dh)
+        ## Update the shape function gradients based on the cell coordinates
+        reinit!(cellvalues, cell)
+        ## Reset the element stiffness matrix
+        fill!(ke, 0.0)
+        ## Compute element contribution
+        assemble_cell!(ke, cellvalues, ∂σ∂ε)
+        ## Assemble ke into K
+        assemble!(assembler, celldofs(cell), ke)
+    end
+    return K
+end
+#md nothing #hide
+
+# ### Solution of the system
+# The last step is to solve the system. First we allocate the global stiffness matrix `K`
+# and assemble it.
+K = allocate_matrix(dh)
+assemble_global!(K, dh, cellvalues, E4);
+# Then we allocate and assemble the external force vector.
+f_ext = zeros(ndofs(dh))
+assemble_external_forces!(f_ext, dh, getfacetset(grid, "top"), facetvalues, traction);
+
+# To account for the Dirichlet boundary conditions we use the `apply!` function.
+# This modifies elements in `K` and `f`, such that we can get the
+# correct solution vector `u` by using solving the linear equation system $K_{ij} \hat{u}_j = f^\mathrm{ext}_i$,
+apply!(K, f_ext, ch)
+u = K \ f_ext;
+
+# ### Exporting to VTK
+# To visualize the result we export the grid and our field, `u`,
+# to a VTK-file, which can be viewed in e.g. [ParaView](https://www.paraview.org/).
+# For fun we'll color the logo, and thus create cell data with numbers according
+# to the logo colors.
+color_data = zeros(Int, getncells(grid))
+colors = Dict(
```

Comment:
The Julia logo colors are cute but I wonder if not showing how to output the stress or strain would be more instructive for your typical FEM beginner.

Reply:
Hid this now then, and added stress calculation and export as both cell data and L2Projection.